PolymerComponent.js

```diff
@@ -166,6 +176,11 @@ class PolymerComponent extends React.Component {
 
         }
 
+        // classes
+        if (props.className) {
```

Comment:
Is this condition still needed? If a class name is added and then removed from the props then it won't get removed here. For example:

1. `className` is initialized to `undefined`, which means that no class is added
1. `className` is set to `'red'`, which means this conditional runs add the class `red` is added
1. `className` is set back to `undefined`, meaning this conditional is _not_ run and the class name remains `red`.

I believe I have that right. In the `className` test maybe we should check what happens if the class name is removed after adding it?

Reply:
i don't think it is needed anymore. I will remove it.